riscv-executor/src/lib.rs

```diff
@@ -422,7 +555,8 @@ mod builder {
         ///
         /// to set the PC, use set_pc() instead of this
         pub(crate) fn set_reg(&mut self, idx: &str, value: impl Into<Elem<F>>) {
-            self.set_reg_impl(idx, value.into())
+            let v = value.into();
+            self.set_reg_impl(idx, v)
```

Comment:
why the change?

Reply:
prob. leftover from some "println debugging", ill revert